### PR TITLE
Require First Pixel Extension as 0 [minor]

### PR DESCRIPTION
**Pull request by ric-evans**
Now, the first pixel extension number must be `0`. This is really a placeholder since `all pixels over the sky are scanned` in the first iteration. In actuality, this could be forced to any number -- talking with @tianluyuan, we settled on `0` since this will make it evident that the number is unique.

Previously, the first pixel extension number had to be `12`. This was due to my misunderstanding of the process at the time.

### CODE REVIEW

**Review by tianluyuan — APPROVED** [review]
Yes better to make this its own PR. LGTM

File: skymap_scanner/server/utils.py [hunk 1]
```diff
@@ -159,20 +164,31 @@ def __hash__(self) -> int:  # type: ignore[override]
...
     def n_recos_by_nside_lowerbound(self, n_posvar: int) -> Dict[int, int]:
-        """Get estimated # of recos per nside.
+        """Get # of recos per nside (w/ predictive scanning its a LOWER bound).
 
-        These are ESTIMATES (w/ predictive scanning it's a LOWER bound).
+        The actual pixel generation is done iteratively, and does not
+        rely on this function. Use this function for reporting & logging
+        only.
         """
         int_int_list = self._get_int_int_list()
```
Comment (tianluyuan) [inline skymap_scanner/server/utils.py:174]: Would it not be simpler to retain the previous code and just create a new `int_int_list` with `12` in the first tuple?  I'm not sure the additional functions improve the readability, and both are just trying to catch the same case.

**Review by tianluyuan — COMMENTED** [review]
If we wanted to keep the code as before, maybe it would be fine to just keep the default at `12` and instead of referring to it as "pixel extension" just refer to it as "nside factor" or something like that

File: skymap_scanner/server/utils.py (hunk 1 above)
Comment (ric-evans) [reply skymap_scanner/server/utils.py:174]: That's a great point. I didn't think about that

File: skymap_scanner/server/utils.py [hunk 2]
```diff
@@ -159,20 +164,31 @@ def __hash__(self) -> int:  # type: ignore[override]
...
+                return self.HEALPIX_BASE_PIXEL_COUNT
+            return pixext
 
         return {
-            N[0]: int(n_posvar * N[1] * (N[0] / previous_nside(N)) ** 2)
-            for N in int_int_list
+            nside: int(
+                n_posvar
+                * get_npixels_multiplier(i, pixext)
+                * (nside / get_nside_divisor(i))
```
[the remark this answers is not in the document]
Comment (ric-evans) [reply skymap_scanner/server/utils.py]: I believe this is/was right: `nside / previous_nside(i) ** 2)`

[the remark this answers is not in the document]
Comment (ric-evans) [reply skymap_scanner/server/utils.py]: fixed

**Pull request merged**